features/node/nto.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Node Tuning Operator related scenarios
+
+Background: Cleaning before if some previous test run failed
```

Comment:
@skordas you don't need to do this...what are you trying to accomplish?

Reply:
Please check line 34
it will create custom resource (new tuned) which will be removed in line 66
if test will fail (somewhere between) - then any other scenario which is using the same tuned will failed (leftovers from other tuned tests). This is first one, but it will be more :)